components/form.rst

```diff
@@ -750,7 +750,7 @@ method to access the list of errors. It returns a
     $errors = $form['firstName']->getErrors();
 
     // a FormErrorIterator instance in a flattened structure
-    $errors = $form->getErrors(true);
+    $errors = $form->getErrors(false, true);
```

Comment:
I think the previous example is correct. We do want to get the errors from nested forms (that's why the first argument should be `true`), but we want to flatten the structure (the second argument defaults to `true`).

Reply:
I think I got somehow confused by this. I actually had to read the docs many times to get it, I got better help by reading the method signature in the end.
Maybe the comment is wrong, then?

Shouldn't this be sth like:
```
    // a FormErrorIterator instance including child forms in a flattened structure
    $errors = $form->getErrors(true);
    
        // a FormErrorIterator instance including child forms without flattening the output structure
    $errors = $form->getErrors(true, false);
```

Reply:
we could indeed update the comment to what you suggest to clarify what is happening here

Reply:
Done and updated. :)